elasticsearch/includes/classes/class-health-job.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Automattic\VIP\Elasticsearch;
+
+use Automattic\VIP\Elasticsearch\Health as Health;
```

Comment:
I'm not sure if this is really needed since they're both in the same namespace?

Reply:
I added that in because it was giving a class not found error before but you're right, it shouldn't be needed if it's in the same namespace. Must have done something else wrong